pulumi/grapl/__main__.py

```diff
@@ -114,6 +114,10 @@ def main() -> None:
     if not config.LOCAL_GRAPL:
         upstream_stacks = UpstreamStacks()
         nomad_provider = get_nomad_provider_address(upstream_stacks.nomad_server)
+        # Using get_output instead of require_output so that preview passes.
```

Comment:
this comment was preexisting

Reply:
Does it still need to use `get_output` in that case?

Reply:
I don't know if it ever did in the first place tbh, and I don't want to stand up an entire platform-infrastructure stack to test this 😢 (since this only happens if not config.LOCAL_GRAPL)